lib/active_model/serializer/array_serializer.rb

```diff
@@ -24,6 +25,7 @@ def initialize(objects, options = {})
         end
         @meta     = options[:meta]
         @meta_key = options[:meta_key]
+        @pagination = ActiveModel::Serializer::Pagination.new(objects) if options[:pagination]
```

Comment:
nitpick: `PaginationLinks` is probably a better name. 

Also, I think this code should also be in the [json_api adapter](https://github.com/rails-api/active_model_serializers/blob/03ddeeeec1e4eedafeb68eeccd59e8c3826b859d/lib/active_model/serializer/adapter/json_api.rb#L20-L30)